shared/editor/embeds/Grist.tsx

```diff
@@ -22,10 +22,6 @@ function Grist(props: Props) {
   );
 }
 
-Grist.ENABLED = [
-  new RegExp(
-    "^https?://([a-z.-]+\\.)?getgrist\\.com/doc/(.+)$"
-  ),
-];
+Grist.ENABLED = [new RegExp("^https?://([a-z.-]+\\.)?getgrist\\.com/(.+)$")];
```

Comment:
Sorry, why was this changed?

```suggestion
Grist.ENABLED = [new RegExp("^https?://([a-z.-]+\\.)?getgrist\\.com/doc/(.+)$")];
```

Reply:
https://docs.getgrist.com/sg5V93LuAije/Untitled-document/p/22?embed=true

Is a valid URL. "doc" is not required.

Reply:
Cool then. My bad!